serializable.ts

```diff
@@ -4,8 +4,21 @@ import { SerializePropertyOptionsMap } from "./serialize_property_options_map.ts
 import { defaultToJson } from "./to_json/default.ts";
 import { recursiveToJson } from "./to_json/recursive.ts";
 
+/** to be implemented by external authors on their models  */
+export declare interface TransformKey {
```

Comment:
Is the “declare” keyword needed here?

Reply:
yea, without it we do not get the errors to populate if implemented wrong when declared with `implements TransformKey`

Although some more testing around this wouldn't hurt